valhalla/thor/bidirectional_astar.h

```diff
@@ -119,20 +121,37 @@ class BidirectionalAStar : public PathAlgorithm {
    */
   void ExpandForward(baldr::GraphReader& graphreader,
                      const baldr::GraphId& node,
-                     const sif::BDEdgeLabel& pred,
+                     sif::BDEdgeLabel& pred,
                      const uint32_t pred_idx,
                      const bool from_transition);
+  // Private helper function for `ExpandForward`
+  bool ExpandForwardInner(baldr::GraphReader& graphreader,
```

Comment:
have you tried forcing these to be inlined to get back the 15% drop you had from the refactor?

Reply:
I did, did not see any measurable change.